### ISSUE TITLE
Deleting existing share link creates a new one

### ISSUE THREAD

**Issue by schiessle** [nextcloud/desktop#1179]
Tested with the Desktop client v 2.5.1 and Nextcloud 15.0.6

Steps to test:

- create a public link for a folder
- try to delete it again
  - a confirmation client ask if I really want to delete it
  - I chose "yes", the public link checkbox is checked again
  - try this multiple times
- look at the share dialog of the folder in the web view
- you will see that for every try to delete the public link a new one was created

**Comment by camilasan** [comment]
I have fixed it on 2.5.3 rc1:
https://download.nextcloud.com/desktop/prereleases/Linux/Nextcloud-2.5.3.20190412-rc1-x86_64.AppImage
https://download.nextcloud.com/desktop/prereleases/Windows/Nextcloud-2.5.3-rc1-20190412.exe
https://download.nextcloud.com/desktop/prereleases/Mac/Nextcloud-qt5.9.2-2.5.3.20190412rc1.pkg

Could you try it?

**Comment by biva** [comment]
Hello,
Any update on this issue?
Thanks!

**Comment by camilasan** [comment]
Yes, did you try the rc for 2.5.3? The fix is there.

**Comment by biva** [comment]
Tested on Windows with Nextcloud-2.5.3.5524-rc2-20190612.exe: it works, thanks!

But in a production environment, it's not easy to test: when I install the client, even if I chose the custom install in a different location, the new client uses the configuration of the former client: if there is a bug in the tested version, the sync could create issues on production data or production server. Is there a possibility to install a test client totally independent from any previous Nextcloud installation, and set it up as if it was a fresh new install?

If it's not possible, I will not be able to test any other RC or beta version :(
